pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/QueryQuotaClusterIntegrationTest.java

```diff
@@ -319,7 +319,7 @@ private void verifyQuotaUpdate(float quotaQps) {
         } catch (IOException e) {
           throw new RuntimeException(e);
         }
-      }, 5000, "Failed to reflect query quota on rate limiter in 5s.");
+      }, 10000, "Failed to reflect query quota on rate limiter in 5s.");
```

Comment:
Is this change expected?

Reply:
This change is not strictly necessary, so I'll revert it.